docs/Updating-npm-bundled-node-gyp.md

```diff
@@ -1,63 +1,41 @@
 
-So instead:
+The catch is that npm won't use this version unless you tell it to, it'll keep on using the one you have installed. You need to instruct it to by setting the `node_gyp` config variable (which goes into your `~/.npmrc`). You do this by running the `npm config set` command as below. Then npm will use the command in the path you supply whenever it needs to build a native addon.
 
-## Version of npm
+**Important**: You also need to remember to unset this when you upgrade npm with a newer version of node-gyp, or you have to manually keep your globally installed node-gyp to date. See "Undo" below.
 
-We need to start by knowing your version of `npm`:
-```bash
-npm --version
+## Linux and macOS
 ```
-
-## Linux, macOS, Solaris, etc.
-
-Unix is easy. Just run the following command.
-
-If your npm is version ___7___, do:
-```bash
-$ npm explore npm/node_modules/@npmcli/run-script -g -- npm_config_global=false npm install node-gyp@latest
-```
-
-Else if your npm is version ___less than 7___, do:
-```bash
-$ npm explore npm/node_modules/npm-lifecycle -g -- npm install node-gyp@latest
+npm install --global node-gyp@latest
+npm config set node_gyp $(npm prefix -g)/lib/node_modules/node-gyp/bin/node-gyp.js
 ```
 
-If the command fails with a permissions error, please try `sudo` and then the command.
+`sudo` may be required for the first command if you get a permission error.
 
 ## Windows
+@joaocgreis' Windows instructions from [#1753 (comment)](https://github.comnodejs/node-gyp/issues/1753#issuecomment-501827267)
 
-Windows is a bit trickier, since `npm` might be installed to the "Program Files" directory, which needs admin privileges in order to
-modify on current Windows. Therefore, run the following commands __inside a `cmd.exe` started with "Run as Administrator"__:
-
-First we need to find the location of `node`. If you don't already know the location that `node.exe` got installed to, then run:
-```bash
-$ where node
+### Windows Command Prompt
 ```
-
-Now `cd` to the directory that `node.exe` is contained in e.g.:
-```bash
-$ cd "C:\Program Files\nodejs"
+npm install --global node-gyp@latest
+for /f "delims=" %P in ('npm prefix -g') do npm config set node_gyp"%P\node_modules\node-gyp\bin\node-gyp.js"
 ```
 
-If your npm version is ___7___, do:
-```bash
-cd node_modules\npm\node_modules\@npmcli\run-script
+### Powershell
 ```
-
-Else if your npm version is ___less than 7___, do:
-```bash
-cd node_modules\npm\node_modules\npm-lifecycle
+npm install --global node-gyp@latest
+npm prefix -g | % {npm config set node_gyp "$_\node_modules\node-gyp\bin\node-gypjs"}
 ```
 
-Finish by running:
-```bash
-$ npm install node-gyp@latest
+## Undo
+**Beware** if you don't unset the `node_gyp` config option, npm will continue to use the globally installed version of node-gyp rather than the one it ships with,which may end up being newer.
```

Comment:
```suggestion
**Beware** if you don't unset the `node_gyp` config option, npm will continue to use the globally installed version of node-gyp rather than the one it ships with, which may end up being newer.
```